lib/decimal.ex

```diff
@@ -70,8 +72,10 @@ defmodule Decimal do
   @type exponent :: integer
 
   @typedoc """
+
     * `1` for positive
     * `-1` for negative
+
```

Comment:
@ericmj we should add it to a "documentation" style guide which still needs to happen 😄